hawkbit-simple-ui/src/main/java/org/eclipse/hawkbit/ui/simple/view/TargetView.java

```diff
@@ -239,6 +243,13 @@ private void setItem(final MgmtTarget target) {
             lastModifiedBy.setValue(target.getLastModifiedBy());
             lastModifiedAt.setValue(new Date(target.getLastModifiedAt()).toString());
             securityToken.setValue(target.getSecurityToken());
+            var response = hawkbitClient.getTargetRestApi().getAttributes(target.getControllerId());
+            if (response.getStatusCode().is2xxSuccessful()) {
+                targetAttributes.setValue(response.getBody().entrySet().stream().map(entry -> entry.getKey()+": "+
```

Comment:
If your target doesn't have any attributes the body is null and throws exception at this line:
```
java.lang.NullPointerException: Cannot invoke "org.eclipse.hawkbit.mgmt.json.model.target.MgmtTargetAttributes.entrySet()" because the return value of "org.springframework.http.ResponseEntity.getBody()" is null
	at org.eclipse.hawkbit.ui.simple.view.TargetView$TargetDetails.setItem(TargetView.java:248) ~[classes/:na]
```
please handle response.getBody() if null

Reply:
handled